task4/HelloWorld.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+import (
+  "github.com/kyokomi/emoji"
```

Comment:
This repo should contain `Gopkg.{toml,lock}` or `go.{mod,sum}` files depending on system used